apps/docs/src/app/platform/component-docs/platform-menu-button/platform-menu-button-header/platform-menu-button-header.component.html

```diff
@@ -1,8 +1,19 @@
 <header>Menu Button</header>
 <description>
-    Menu Button works as Dropdown.
-    <br />
-    It selects from Menu whichs appears on Menu button click.
+    The menu button is used for triggering the display of a menu. <br />
+    In ngx-core, it is an extenstion of ngx-button directive, but in Platform this is a separate component.
```

Comment:
Could you say as to why it is different

Reply:
i have added "it reduces the complexity to create Menu Button." . also in bullet points it is explained. now it looks more specific.